### DISCUSSION TITLE
Mounting the headnode

### DISCUSSION THREAD

**Discussion by 15kay** [chpc-tech-eval/scc#163] [General]
Hey there

can anyone please help me by  successfully mounting the headnode....I'm definitely going to end up in hospital because of this error
![image](https://github.com/user-attachments/assets/5a98c1aa-3d29-4a4c-817e-162c1b4b3c6b)

**Comment by 15kay** [comment]
it is hanging when i mount
![image](https://github.com/user-attachments/assets/4d8ea770-3b6f-4c65-b290-cc5dde9f48ec)

**Comment by mmhashatsi** [comment]
Please check that the /etc/fstab file is on the correct node
